docs/docs/0.3.2/documentation/README.md

```diff
@@ -71,6 +71,32 @@ When following the installation instructions, `kuma-injector` will be automatica
 <img src="/images/docs/0.2.0/diagram-08.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
+#### Matching Labels in `Deployment` and `Service`
```

Comment:
Sorry, I think that this information is too specific to a particular user from Kuma Slack who made a typo in `Service` or `Deployment`.

I expect that the majority of `k8s` users already understand how a `Service` matches a `Pod`.

So, let's rephrase the text to focus on the part that `Kuma` requires every `Pod` to be a part of at least one `Service`.

It will be shorter and relevant to everyone.

Reply:
@devadvocado can you assist with the verbiage for this title change?